pkg/routine/config.go

```diff
@@ -0,0 +1,22 @@
+// Copyright © 2020 by PACE Telematics GmbH. All rights reserved.
+// Created at 2020/02/27 by Marius Neugebauer
+
+package routine
+
+import (
+	"time"
+
+	"github.com/caarlos0/env"
+)
+
+type config struct {
+	RedisLockTTL time.Duration `env:"ROUTINE_REDIS_LOCK_TTL" envDefault:"1s"`
```

Comment:
As discussed in person: We should maybe think about the rate how often the `TTL` gets refreshed. Right now it seems to be `TTL/2` (https://github.com/pace/bricks/pull/171/files#diff-ee9982e919501ff31f7336477af2941bR30). I personally think we should increase this ration to a `TTL/5` or even `TTL/10` which would make it highly unlikely in a realistic scenario, that the current master loses his lock even though he is running fine and just couldn't refresh the lock because of network problems or something else. This would mean that we also should think about raising the general `TTL` to `5s` or `10s` to avoid golang runtime specific effects of a too fast refresh rate.